lib/modules/apostrophe-schemas/index.js

```diff
@@ -1589,7 +1599,7 @@ module.exports = {
           safeFor: 'manage',
           launder: joinFilterLaunder
         });
-        
+
```

Comment:
Please submit without your whitespace-only changes in future as they make diffs confusing.